tests/tests_integration/test_api/test_synthetic_time_series.py

```diff
@@ -55,20 +64,20 @@ def test_retrieve_with_errors(self, test_time_series, post_spy):
     def test_expression_builder_time_series_vs_string(self, test_time_series):
         from sympy import symbols
 
-        dps1 = COGNITE_CLIENT.datapoints.synthetic.retrieve(
-            expression=symbols("a"),
+        dps1 = COGNITE_CLIENT.datapoints.synthetic.query(
```

Comment:
please change one of them to single item, one to single item list, one to multi item list and check return types

Reply:
ah, you're right, changed regarding your suggestion